src/components/Menu/config/footerConfig.ts

```diff
@@ -0,0 +1,76 @@
+import { FooterLinkType } from '@pancakeswap/uikit'
+import { ContextApi } from 'contexts/Localization/types'
+
+export const footerLinks: (t: ContextApi['t']) => FooterLinkType[] = (t) => [
+  {
+    label: t('About'),
+    items: [
+      {
+        label: t('Contact'),
+        href: 'https://docs.pancakeswap.finance/contact-us',
+      },
+      {
+        label: t('Blog'),
+        href: 'https://pancakeswap.medium.com/',
+      },
+      {
+        label: t('Community'),
+        href: 'https://docs.pancakeswap.finance/contact-us/telegram',
+      },
+      {
+        label: t('CAKE token'),
+        href: 'https://docs.pancakeswap.finance/tokenomics/cake',
+      },
+      {
+        label: '—',
+      },
+      {
+        label: t('Online Store'),
+        href: 'https://pancakeswap.creator-spring.com/',
+        isHighlighted: true,
+      },
+    ],
+  },
+  {
+    label: t('Help'),
+    items: [
+      {
+        label: t('Customer Support'),
+        href: 'https://docs.pancakeswap.finance/contact-us/customer-support',
```

Comment:
Again not that you asked: These links open a new window, but once you click any of these external links, it keeps the same tab / window as the focus for load. So if I click to open a new window for the docs, but then click to get the store, it doesn't open a new window it just loads the link in the already open one.

To reproduce ctrl + click on store (for new tab / window) then go back to the original window and ctrl + click on another link. It will not open a new tab, it will just load the first one with the new link clicked.

Reply:
https://user-images.githubusercontent.com/1007667/133295017-dd3394eb-4784-4165-93cf-491b0af07324.mov

Reply:
Good point, this will be fixed on next UIKit release: https://github.com/pancakeswap/pancake-toolkit/pull/278